charts/budibase/templates/app-service-deployment.yaml

```diff
@@ -152,6 +152,8 @@ spec:
           value: {{ .Values.globals.tenantFeatureFlags | quote }}
         - name: ENCRYPTION_KEY
           value: {{ .Values.globals.bbEncryptionKey | quote }}
+        - name: TEMP_BUCKET_NAME
+          value: {{ .Values.globals.tempBucketName | quote }}
```

Comment:
This is more about me being picky but could we move this close to the rest of the buckets?

https://github.com/Budibase/budibase/blob/e901ea8a46b163d233abd1d082d93d146eb24269/charts/budibase/templates/app-service-deployment.yaml#L101

It makes more sense to have them all grouped, just because we can search faster and use them as in-code docs.

Reply:
Done!